src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<FruitTransaction> fruits = new ArrayList<>();
```

Comment:
**fruits** and **Transaction** are different things. Fruits are kept in the Storage in Fruit Shop.
```suggestion
    public static final Map<String, Integer> fruits = new HashMap<>();
```